apps/cbportal/base/cloudbroker__grid/methodclass/cloudbroker_grid.py

```diff
@@ -1,11 +1,14 @@
 from JumpScale import j
 from JumpScale.portal.portal.auth import auth
 from JumpScale.portal.portal import exceptions
+import requests
+improt time
 
 class cloudbroker_grid(object):
 
     def __init__(self):
-        self.models = j.clients.osis.getNamespace('cloudbroker')
+        self.cbmodels = j.clients.osis.getNamespace('cloudbroker')
```

Comment:
for some reason this actor doesnt inherit from actorbase.
Please change this what way self.models will be available.
Also i dont unnderstand why you renamed it everywhere in the cloudbroker/cloudapi actors self.models exists and is the cloudbroker namespace